Tests/verify_end_to_end_purchase.py

```diff
@@ -0,0 +1,175 @@
+import time
+import unittest
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service as ChromeService
+from webdriver_manager.chrome import ChromeDriverManager
+
+from Pages.home_page import HomePage
+from Pages.login_page import LoginPage
+from Pages.my_account_page import MyAccountPage
+from Pages.product_page import ProductPage
+from Pages.cart_page import CartPage
+from Pages.checkout_page import CheckOutPage
+
+
+class TestEndToEndPurchase(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.browser = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
+        cls.checkout_page = CheckOutPage(cls.browser)
+
+    def test_end_to_end_purchase(self):
+        browser = self.browser
+
+        browser.get("http://demostore.supersqa.com/")
+
+        # Login to account
+        homepage = HomePage(browser)
+        homepage.click_myAccount()
+
+        login = LoginPage(browser)
+        login.enter_username('shopper@gmail.com')
+        login.enter_password('MyPassword1234$')
+        login.click_login()
+
+        time.sleep(2)
```

Comment:
I would recommend using a WebDriverWait function here. It's not good to use time.sleep when automating

Reply:
WebDriverWait replaces time.sleep where necessary.